### PR TITLE
DR-1335 Move layout stuff out of routing layer

### PR DESCRIPTION
**Pull request by myessail**
Keeping some layout stuff in `Private.jsx` because those tabs are specific to logged-in folks, but this takes care of the weird magic numbers and makes the divs all have the correct (intuitive) heights.

### CODE REVIEW

File: src/containers/App.jsx [hunk 1]
```diff
@@ -22,11 +22,11 @@ import Private from 'routes/Private';
 import NotFound from 'routes/NotFound';
 import Logo from 'components/Logo';
 
-import { logOut } from 'actions/index';
 import RoutePublic from 'components/RoutePublic';
 import RoutePrivate from 'components/RoutePrivate';
 import { ReactComponent as CarrotSVG } from 'media/icons/angle-line.svg';
 import { ReactComponent as SignOutSVG } from 'media/icons/logout-line.svg';
+import { logOut } from 'actions';
```
Comment (RoriCremer) [inline src/containers/App.jsx:30]: how was this working before?

**Review by RoriCremer — APPROVED** [review]

File: public/index.html [hunk 2]
```diff
@@ -23,254 +23,256 @@
...
+          body {
+            background: linear-gradient(to bottom, #00b4d5, #00191e);
           }
-        }
 
-        @-webkit-keyframes animate-to-3 {
-          25%, 75% {
-            -webkit-transform: translateX(1.5rem) scale(0.75);
-            transform: translateX(1.5rem) scale(0.75);
+          .blob {
```
Comment (nmalfroy) [inline public/index.html:55]: It feels strange that we could keep CSS in the index.  Feels like this is stuff that should live in a root component

**Review by nmalfroy — APPROVED** [review]
Looks good.  Eventually, I'd love to move the css out of index.html and into a root component

File: src/containers/App.jsx [hunk 3]
```diff
@@ -125,7 +126,9 @@ const useStyles = makeStyles((theme) => ({
   },
   content: {
     flexGrow: 1,
-    height: '100vh',
+    paddingTop: 64,
```
Comment (fboulnois) [inline src/containers/App.jsx:129]: 👍

**Review by fboulnois — APPROVED** [review]
lgtm

**Pull request merged**